apollo-core/src/main/java/com/ctrip/framework/foundation/internals/Utils.java

```diff
@@ -16,18 +16,27 @@
  */
 package com.ctrip.framework.foundation.internals;
 
-import com.google.common.base.Strings;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
 
 public class Utils {
+
+  /**
+   * Judge if the given string contains any char.
+   *
+   * @param str
+   * @return true if the given string is blank
+   */
   public static boolean isBlank(String str) {
-    return Strings.nullToEmpty(str).trim().isEmpty();
+    return StringUtils.isBlank(str);
   }
 
+  /**
+   * Judge if the system is windows.
+   *
+   * @return true if the current system is windows.
+   */
   public static boolean isOSWindows() {
-    String osName = System.getProperty("os.name");
```

Comment:
Yes, the implementation is the same, so this pr is not urgent. I just think it would be more elegant to use the existing utils, if we plan to upgrade commons-lang to commons-lang3.

As you said, it's unnecessary to extra dependency to apollo-core, I very agree with you, so I remove these changes.